dev/data/dashboard.json

```diff
@@ -2,202 +2,1513 @@
   {
     "kind": "Dashboard",
     "metadata": {
-      "name": "SimpleLineChart",
-      "project": "perses"
+      "name": "DemoDashboard",
+      "project": "perses",
+      "created_at": "2021-11-09T00:00:00Z",
+      "updated_at": "2021-11-09T00:00:00Z",
+      "version": 0
     },
     "spec": {
-      "datasource": {
-        "name": "PrometheusDemo",
-        "kind": "Prometheus",
-        "global": true
-      },
-      "duration": "6h",
-      "variables": {
-        "labelName": {
-          "kind": "LabelNamesQuery",
-          "displayed_name": "LabelName",
-          "hide": false,
-          "parameter": {
-            "matchers": [
-              "up"
-            ],
-            "capturing_regexp": "(.*)"
-          }
-        },
-        "labelValue": {
-          "kind": "LabelValuesQuery",
-          "displayed_name": "LabelValue",
-          "hide": false,
-          "parameter": {
-            "label_name": "$labelName",
-            "matchers": [
-              "up"
-            ],
-            "capturing_regexp": "(.*)"
+      "duration": "5m",
+      "variables": [
+        {
+          "kind": "ListVariable",
+          "spec": {
+            "name": "job",
+            "allow_multiple": true,
+            "allow_all_value": true,
+            "default_value": "node",
+            "plugin": {
+              "kind": "PrometheusLabelValuesVariable",
+              "spec": {
+                "label_name": "job"
+              }
+            }
+          }
+        },
+        {
+          "kind": "ListVariable",
+          "spec": {
+            "name": "instance",
+            "allow_all_value": true,
+            "plugin": {
+              "kind": "PrometheusLabelValuesVariable",
+              "spec": {
+                "label_name": "instance",
+                "matchers": [
+                  "up{job=\"$job\"}"
+                ]
+              }
+            }
+          }
+        },
+        {
+          "kind": "ListVariable",
+          "spec": {
+            "name": "interval",
+            "plugin": {
+              "kind": "StaticListVariable",
+              "spec": {
+                "values": [
+                  "1m",
+                  "5m"
+                ]
+              }
+            }
           }
         }
-      },
+      ],
       "panels": {
```

Comment:
Does anything need to be updated in CONTRIBUTING.md about local development, sounds like we just update this file, reload, and we're good to go?

Reply:
So hopefully I have already updated correctly this document.

Actually this file is just used as a based to generate the database. Once you have ran the script `populate.sh` you have a list of folder that appears in the `dev` folder like this: 
<img width="324" alt="image" src="https://user-images.githubusercontent.com/4548045/196766513-724d24ae-c805-41fb-a6d7-be572bb77c85.png">

Then if you modify any files in the folder `local_db` it will then be reflected in the UI without restarting the backend.

Updating the file `dev/data/dashboards.json` should occur only if you want to persist some modification to the dashboard Demo / Benchmark or you want to add a new dashboard.

Let me know if you think it needs more explication in the doc `CONTRIBUTING.md` :)